_includes/subscribe-modal.html

```diff
@@ -0,0 +1,44 @@
+<div class="modal fade" id="subscribe-success" tabindex="-1" role="dialog" aria-hidden="true">
+    <div class="modal-dialog modal-dialog-centered subscribe-modal " role="document">
+        <div class="modal-content text-center">
+            <div class="modal-header">
+                <button type="button" class="close" onclick="window.history.pushState({path: window.location.pathname },'',window.location.pathname)"  data-dismiss="modal" aria-label="Close">
```

Comment:
I'm having a brain fart... Can you remind me why we do a `pushState` on the modal?

Reply:
To remove the parameter from the route which in this case is `subscribe_modal = 1` and avoid reloading that page

Reply:
Ah, right. Sorry for forgetting. Perhaps add a comment with that explanation?